packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -110,6 +97,8 @@ function pascalCase(str: string) {
   return camelCase[0].toUpperCase() + camelCase.substr(1);
 }
 
+// Existing high cyclomatic complexity
```

Comment:
Left on rules to give an aggressive warning for functions with a high cyclomatic complexity. This got flagged for a few 200 line monster functions that really need some refactoring, so the defaults seem good to me. The goal is to encourage folks adding new complexity to refactor as needed.